vXXX/content/auswertung.tex

```diff
@@ -8,5 +8,23 @@ \section{Auswertung}
   \label{fig:plot}
 \end{figure}
 
+\begin{table}
+  \centering
+  \caption{Eine Beispieltabelle mit Messdaten.}
+  \label{tab:tabelle}
+  \sisetup{table-format=1.2}
+  \begin{tblr}{
+      colspec = {S[table-format=3.0] S S S S[table-format=3.2]},
+      row{1} = {guard, mode=math},
+    }
+    \toprule
+    f & l_\text{start} & l_1 & l_{\text{kor},1} & B_1 \\
```

Comment:
No units?

Reply:
Ah, right, I just copied on of the examples from the latex slides :D I'll create a more suitable table